SYCL/ESIMD/api/functional/ctors/ctor_load_acc_core.cpp

```diff
@@ -47,12 +47,12 @@ int main(int, char **) {
   const auto offset_generators = ctors::coverage::get_all_offset_generators();
 
   // Run over all types with element and vector alignment
-  // Default tag for overaligned is configured in a way appropriate for any
-  // type from the type coverage, so run over it as well
+  // Using overaligned tag with alignment requirements appropriate for any
+  // type from the type coverage
   {
     const auto alignments =
         named_type_pack<ctors::alignment::element, ctors::alignment::vector,
-                        ctors::alignment::overal<>>::generate();
+                        ctors::alignment::overal<16>>::generate();
```

Comment:
please replace 16 with named constant - `esimd::OperandSize::OWORD`

Reply:
It would be `esimd::detail` namespace, not `esimd`. Usually it's not good to rely on implementation details consistence, as it might result in less flexibility for product development. Should we trigger test failure in case of `esimd::detail::OperandSize` renaming or header accessibility?

I might provide a `constexpr oword = 16` in each test, but `octoword` should always be `16 bytes` by definition. What do you think?

Reply:
@kbobrovs Friendly ping on minor question

Reply:
constexpr oword = 16 LGTM

Reply:
Thanks, applied by 029e62a90d2bb5c2ab0c7b06f71c74ee285dd6e0